crates/transaction-emitter-lib/src/args.rs

```diff
@@ -133,6 +134,15 @@ pub struct EmitArgs {
     #[clap(long, min_values = 0)]
     pub transaction_phases: Vec<usize>,
 
+    #[clap(long)]
+    pub gas_price: Option<u64>,
+
+    #[clap(long)]
+    pub max_gas_per_txn: Option<u64>,
+
+    #[clap(long)]
+    pub init_gas_price_multiplier: Option<u64>,
```

Comment:
nit: Since this is only being used in minting, should we rename this to `mint_gas_price_multiplier` for clarity?

Reply:
no, this is used for all initialization - things like publishing modules for transactions types that need it, etc.

Reply:
fixed this, so now it is used for all initialization